src/ducks/classrooms.js

```diff
@@ -141,7 +141,7 @@ Effect('getClassroomsAndAssignments', (selectedProgram) => {
         // loop through the number of pages to request all of the data
         // and concatenate the response data together for the app state
         // Neither Pagination nor infinite scroll would be good UX for current table design.
-        Actions.getAssignments(classroom.id);
+        Actions.getAssignments({ classroomId: classroom.id, selectedProgram });
```

Comment:
Out of curiosity, is there any reason the function `getAssignments()` is structured like `getAssignments(data)` where `data = {classroomId, selectedProgram}` instead of `getAssignments(classroomId, selectedProgram)`?

It's functionally the same, but from a readability standpoint, it's not very clear what arguments/values `getAssignments()` is expecting.

The only advantages that I can see for using a single 'compound variable' instead of spelling out the different variables are if:
- the compound variable (`data`) supports different kinds of key-values (e.g. an advanced search query, `data = {color:'red', species:'dragon', optionalAge: null }` ), or
- the `data` is structured in a specific way that doesn't matter to the function(), but matters to the API (e.g. createAssignnments())

Reply:
Jumpstate's Effect either wasn't designed to work or had a bug when I tried to pass it two separate arguments! 😢

Reply:
Oh gods, that's right! I totally forgot about Jumpstate's "one Effect, one argument" rule. 🤦‍♂️